packages/basebridge/src/test-transactions.ts

```diff
@@ -0,0 +1,374 @@
+    'bridge eth from base to ethereum',
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge eth from base to ethereum, when contractAddress not defined',
+    { contractAddress: undefined },
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge eth from base to ethereum, when tokenAddress not defined',
+    { tokenAddress: undefined },
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge eth from base to ethereum, when tokenAddress and contractAddress not defined',
+    { tokenAddress: undefined, contractAddress: undefined },
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge eth from base to ethereum, when recipient not defined',
+    { recipient: undefined },
+  ),
+  createTestCase(
+    ETH_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge eth from base to ethereum, when amount not defined',
+    { amount: undefined },
+  ),
+  // erc20 from base
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'when bridge erc20 from base to ethereum',
+  ),
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge erc20 from base to ethereum, when contractAddress not defined',
+    { contractAddress: undefined },
+  ),
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge erc20 from base to ethereum, when tokenAddress not defined',
+    { tokenAddress: undefined },
+  ),
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge erc20 from base to ethereum, when tokenAddress and contractAddress not defined',
+    { tokenAddress: undefined, contractAddress: undefined },
+  ),
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge erc20 from base to ethereum, when recipient not defined',
+    { recipient: undefined },
+  ),
+  createTestCase(
+    ERC20_FROM_BASE_TO_ETHEREUM_TEST,
+    'bridge erc20 from base to ethereum, when amount not defined',
+    { amount: undefined },
+  ),
+]
+
+export const failingTestCases = [
+  // eth from ethereum
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
+    'eth from ethereum to base, when sourceChainId is not correct',
+    {
+      sourceChainId: 5,
+    },
+  ),
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
+    'eth from ethereum to base, when amount is not correct',
+    {
+      amount: '4000000',
+    },
+  ),
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
+    'eth from ethereum to base, when tokenAddress is not correct',
+    {
+      tokenAddress: '0x990D07a69f23d69618e3Bb2444fA07f499193Fe4',
+    },
+  ),
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
+    'eth from ethereum to base, when contractAddress is not correct',
+    {
+      contractAddress: '0x990D07a69f23d69618e3Bb2444fA07f499193Fe4',
+    },
+  ),
+  createTestCase(
+    ETH_FROM_ETHEREUM_TO_BASE_TEST,
+    'eth from ethereum to base, when contractAddress is not correct2',
+    {
+      contractAddress: ETHEREUM_ERC20_BRIDGE_ADDRESS,
```

Comment:
Would just use a totally different contract address here

Reply:
Actually, would just remove this one also. The one above it (ETH_FROM_ETHEREUM_TO_BASE_TEST) should be sufficient.